doc/examples/denoise_ascm.py

```diff
@@ -0,0 +1,184 @@
+"""
+==============================================================
+Denoise images using Adaptive Soft Coefficient Matching (ASCM)
+==============================================================
+
+The adaptive soft coefficient matching (ASCM) as described in [Coupe11]_ is a
+improved extension of non-local means (NLMEANS) denoising. ASCM gives a better
+denoised images from two standard non-local means denoised versions of the
+original data with different degrees sharp feature preserved. Here, one
```

Comment:
Replace this with '... original data with different degrees of sharpness. Here, one... '